### ISSUE TITLE
Typescript error in plugin

### ISSUE THREAD

**Issue by ilyub** [sweepline/eslint-plugin-unused-imports#31]
Getting error for this code:
```ts
import * as _ from "lodash";
```
Error occurs only when the file contains single line.

```
TypeError: Cannot read property 'loc' of null
Occurred while linting ...\tests\index.ts:1
    at Object.problem.fix (...\node_modules\eslint-plugin-unused-imports\lib\rules\predicates.js:57:38)
    at normalizeFixes (...\node_modules\eslint\lib\linter\report-translator.js:193:28)
    at ...\node_modules\eslint\lib\linter\report-translator.js:364:49
    at Object.report (...\node_modules\eslint\lib\linter\linter.js:920:41)
    at Object.value [as report] (...\node_modules\eslint-rule-composer\lib\rule-composer.js:147:31)
    at Program:exit (...\node_modules\@typescript-eslint\eslint-plugin\dist\rules\no-unused-vars.js:302:33)
    at ...\node_modules\eslint\lib\linter\safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (...\node_modules\eslint\lib\linter\safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (...\node_modules\eslint\lib\linter\node-event-generator.js:293:26)
```

**Comment by sweepline** [comment]
Thanks, I guess i need to be smarter about removing newlines.

**Comment by komerdoor** [comment]
Run into the same issue

**Comment by laleksiunas** [comment]
Getting the same error when creating a new file using VS Code/Webstorm and inserting single/multiple import statements. The error goes away after inserting some other expression/statement.

The issue is that there is no validation for the absence of the next token: https://github.com/sweepline/eslint-plugin-unused-imports/blob/master/lib/rules/predicates.js#L57. Not sure what is the expected behavior here: removing all imports from the file or keeping the file as is?

**Comment by sweepline** [comment]
This should be fixed by #34, can you confirm using v1.1.2?

**Comment by laleksiunas** [comment]
I can no longer reproduce the issue. Thanks!
